public/stylesheets/app/_header.scss

```diff
@@ -9,3 +9,7 @@
   cursor: pointer;
   text-decoration: underline !important;
 }
+
+.heading-count-order{
+  color: #505a5f;
```

Comment:
You should use the helper function to return the relevant colour, in this case this would be `govuk-color("dark-grey");`

See: https://design-system.service.gov.uk/styles/colour/